src/main/java/org/reaktivity/nukleus/maven/plugin/internal/ast/parse/AstParser.java

```diff
@@ -446,55 +432,76 @@ public AstNode visitInt8_type(
     public AstNode visitUint64_type(
         Uint64_typeContext ctx)
     {
-        memberBuilder.type(AstType.UINT64).unsignedType(AstType.INT64);
+        if (memberBuilder != null)
+        {
+            memberBuilder.type(AstType.UINT64).unsignedType(AstType.INT64);
+        }
```

Comment:
Are these changes still necessary? Suspect not, now that you have introduced `EnumVisitor` instead.